gcloud/connection.py

```diff
@@ -292,12 +292,13 @@ def api_request(self, method, path, query_params=None,
         if not 200 <= response.status < 300:
             raise make_exception(response, content)
 
-        if content and expect_json:
+        if isinstance(content, six.binary_type):
+            content = content.decode('utf-8')
```

Comment:
@tseaver comment from f2a9c7dfa5f753fc007a8cea4741aa99131da5bb:

> We should not be decoding without looking at the 'Content-Type:' header, which might specify an explicit encoding, or else imply one (e.g., 'application/json' implies 'UTF-8').

----

REPLY: So I should also move the `content_type` check up here as well?

The main reason this code moved from the `if expect_json` block to here is because I wanted it to ignore the `_FutureDict` objects that passed through when `expect_json==True`.